zulipterminal/ui.py

```diff
@@ -53,14 +57,20 @@ def right_column_view(self) -> Any:
         )
 
     def get_random_help(self) -> List[Any]:
-        # Get a hotkey randomly from KEY_BINDINGS
-        random_int = random.randint(0, len(KEY_BINDINGS) - 1)
-        hotkey = list(KEY_BINDINGS.items())[random_int]
-        return [
-            'Help(?): ',
-            ('code', ' ' + ', '.join(hotkey[1]['keys']) + ' '),
-            ' ' + hotkey[1]['help_text'],  # type: ignore
-        ]
+        # Get an allowed (ie. elligible for being displayed as a tip)
+        # hotkey randomly from KEY_BINDINGS
+        l = len(KEY_BINDINGS)
+        random_int = random.randint(0, l - 1)
```

Comment:
I'm not sure if you want this in the same commit (whether as now, or separate as I indicated in the previous review comment), but `random.choice` might be a useful simplification over the current code here.